### ISSUE TITLE
Update supermarket cookbook version.

### ISSUE THREAD

**Issue by jarrettj** [sous-chefs/redisio#355]
Hi,

When using knife cookbook install the version is old. Could you maybe update it to include the CHef 13 compatibility changes? Thanks.

Regards.
JJ

**Comment by tas50** [comment]
This would be really helpful so we could unblock those trying to upgrade from Chef 12 -> Chef 13

**Comment by jarrettj** [comment]
Yip, I take it it is not something easy to do. :( Guess I'll just have to wait.

**Comment by brianbianco** [comment]
The main issue is that I got a new computer and no longer actively use chef.  I just haven't had a chance to get my local environment set up to be able to create the zip, bundle it and upload it.  There is some minimal git work to do that is related to it as well.

I will try to get this done, unfortunately I have a lot of other things competing for my time.  in the meantime you can use one of the several apps to just use the specific REF of the cookbook you'd like to be using.

**Comment by atheiman** [comment]
@brianbianco if you'd like we can move this to the @sous-chefs organization, im sure they'd be happy to take the ownership.

also, there is [a tool called `stove`](https://github.com/sethvargo/stove) which will push to the supermarket in one command

If you want some help with ownership, I can help out.

**Comment by brianbianco** [comment]
@atheiman @sous-chefs Seems reasonable to me that it might be best managed by them.

**Comment by gsdevme** [comment]
Is there any movement here? or is this cookbook just EoL?

**Comment by Xorima** [comment]
Closing due to inactivity. 

If this is still an issue please reopen or open another issue. Alternatively drop by the #sous-chefs channel on the [Chef Community Slack](http://community-slack.chef.io/) and we'll be happy to help! 

Thanks,
Sous-Chefs
